proposals/0345-android-12-issues.md

```diff
@@ -314,7 +460,7 @@ If we decide to use `PendingIntents` and export the `SdlService` we could also c
 
 ### Bluetooth Runtime Permissions
 
-In the use case of the device only being connected over USB, we could try to start the `SdlRouterService` of an application that does have Bluetooth permissions to ensure we select a `SdlRouterService` that can start the `bluetoothTransport`. The problem with this solution is when we choose to start a `SdlRouterService` that is different than the one started by the `USBAccessoryAttachmentActivity`, the `USBTransferProvider` will try to bind to the wrong `RouterService`. This will require a larger refactor of the `USBTransferProvider`.
+In the use case of the device only being connected over USB, we could try to start the `SdlRouterService` of an application that does have Bluetooth permission to ensure we select a `SdlRouterService` that can start the `bluetoothTransport`. The problem with this solution is when we choose to start a `SdlRouterService` that is different than the one started by the `USBAccessoryAttachmentActivity`, the `USBTransferProvider` will try to bind to the wrong `RouterService`. This will require a larger refactor of the `USBTransferProvider`.
```

Comment:
```suggestion
In the use case of the device only being connected over USB, we could try to start the `SdlRouterService` of an application that does have the Bluetooth permission to ensure we select an `SdlRouterService` that can start the `bluetoothTransport`. The problem with this solution is when we choose to start an `SdlRouterService` that is different than the one started by the `USBAccessoryAttachmentActivity`, the `USBTransferProvider` will try to bind to the wrong `RouterService`. This will require a larger refactor of the `USBTransferProvider`.
```